GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1404,7 +1403,7 @@ void OnRevisionReadCompleted()
                         // (can occur when filtering or limiting number of loaded commits)
                         // Try to find the revision where to attach the artificial, first is not found
                         // null means that no matching should be done (just add), so use an empty list if no parents were found
-                        refresh = AddArtificialRevisions(headParents ?? Array.Empty<ObjectId>());
+                        _ = AddArtificialRevisions(headParents ?? []);
```

Comment:
just ignore the return value? (or edit other occurrence)

Reply:
The return value is still used at other callsites.

Reply:
Sharing the point of view of @gerhardol, no need to affect the return value except if it has a special meaning...